### ISSUE TITLE
Add reference field for affiliated registry

### ISSUE THREAD

**Issue by bsipocz** [astropy/astropy.github.com#231]
It would be great to have a field for the preferred reference for the affiliated packages so it's easier for users to figure out what to cite.

**Comment by pllim** [comment]
@lwasser , does pyOpenSci listing come with citation stuff?

**Comment by lwasser** [comment]
@pllim @bsipocz  we capture both the joss doi and a zenodo doi if users provide that . it's not on the website yet but I have a plan to add it. Is that what you mean? Doi links in our yaml file?

**Comment by pllim** [comment]
> Doi links in our yaml file?

I think so... but I will let Brigitta confirm.

**Comment by bsipocz** [comment]
yes, doi links to papers and/or zenodo releases. Please note that for many packages are published as ApJ or similar rather than JOSS, so the field should require some flexibility.
